src/pages/en/guides/content-collections.mdx

```diff
@@ -79,7 +79,20 @@ To benefit from the full TypeScript and autocompletion features of [using schema
   </Fragment>
 </TypeScriptSettingTabs>
 
+### Set up the `.astro` directory
 
+Astro generates TypeScript types from your content collections in the `.astro` directory. These types will be updated anytime you run the [`astro dev`](/en/reference/cli-reference/#astro-dev), [`astro build`](/en/reference/cli-reference/#astro-build), or [`astro sync`](/en/reference/cli-reference/#astro-sync) commands.
+
+:::tip
+If you're using Git for version control, we recommend ignoring this generated directory by adding `.astro` to your `.gitignore`.
+:::
+
+These commands will also generate a [reference path](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-path-) to include `.astro` types in your project. The following will be added to your project's [`src/env.d.ts` file](/en/guides/typescript/#setup) (Astro creates this file if none exists):
```

Comment:
Great minds, Ben!

My only nits are that "Astro creates this file if none exists." is a full sentence and *probably* needs a period? closely followed by a colon? (And, I would have liked a structure where the colon comes immediately after the sentence that says "the following..."

There are ways around that, but I'll consider it non-blocking for now! If it bothers me enough, I can always revisit punctuation/sentence strucure another time!